### PR TITLE
Remove DevCon23 banner

### LINKED ISSUE #9: Remove DevCon23 banner
Remove DevCon23 banner

### PR DESCRIPTION
**Pull request by pedrominatel**
Remove the DevCon23 banner

Closes #9

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -5,19 +5,6 @@ Espressif supports a large variety of open-source projects, including SDKs, comp
...
-We would like to invite you to our second edition of Espressif Developer Conference which will take place on September 12-13, 2023.
-
-Conference is for free and will be streamed on our [Espressif Youtube channel](https://www.youtube.com/@EspressifSystems).
-
-**You can look forward for various IoT topics, more details with full program is available at https://devcon.espressif.com/.**
-
----
-
 <details>
   <summary>For more information about Espressif organization and it's projects click here.</summary>
```
Comment (VojtechBartoska) [inline profile/README.md:9]: @pedrominatel you need to remove also this line to have other content visible by default.

**Review by VojtechBartoska — CHANGES_REQUESTED** [review]
Thanks Pedro for reminder, please update the PR up to the comment and we can merge.

File: profile/README.md (hunk 1 above)
Comment (pedrominatel) [reply profile/README.md:9]: Ok, tks!

**Comment by pedrominatel** [comment]
PTAL @VojtechBartoska.

**Pull request merged**